aerogear-android-push/src/main/java/org/jboss/aerogear/android/unifiedpush/RegistrarManager.java

```diff
@@ -36,18 +35,18 @@
  */
 public class RegistrarManager {
 
-    private static final Map<String, PushRegistrar> registrars = new HashMap<String, PushRegistrar>();
+    private static final Map<String, PushRegistrar> REGISTRARS = new HashMap<>();
 
-    private static final List<MessageHandler> mainThreadHandlers = new ArrayList<MessageHandler>();
```

Comment:
we could be using diamond type on the ArrayList, `new AarrayList<>()`, like above on `REGISTRARS`